huproxy.go

```diff
@@ -109,7 +111,8 @@ func main() {
 
 	log.Printf("huproxy %s", huproxy.Version)
 	m := mux.NewRouter()
-	m.HandleFunc("/proxy/{host}/{port}", handleProxy)
+	fullUrl := fmt.Sprintf("/%s/{host}/{port}", *url)
+	m.HandleFunc(fullUrl, handleProxy)
```

Comment:
Skip the short term variable, just inline.

Reply:
Thanks for reviewing. Updated as requested.